pyproject.toml

```diff
@@ -19,8 +19,11 @@ readme = 'README.rst'
 requires-python = '>= 3.7'
 license = { file = 'LICENSES/MIT.txt' }
 keywords = ['meson', 'build', 'backend', 'pep517', 'package']
-authors = [
-  { name = 'Filipe Laíns', email = 'lains@riseup.net' },
```

Comment:
If we're dropping Filipe's name here, then I think we should give him some credit as original author / emeritus maintainer on https://meson-python.readthedocs.io/en/latest/about.html.

Reply:
Absolutely. Feel free to propose a wording. I'm not particularly good in this kind of things.

Reply:
I suggest changing the Maintainers and Funding section to this:

**Maintainer**

The current maintainers of `meson-python` are:

- [Ralf Gommers](https://github.com/rgommers)
- [Daniele Nicolodi](https://github.com/dnicolodi)
- [Henry Schreiner](https://github.com/henryiii)
- [Thomas Li](https://github.com/lithomas1)

Emeritus maintainers: [Filipe Laíns](https://github.com/FFY00).

**Funding**

Ralf Gommers contributes to the development of `meson-python` as part of their job at [Quansight](https://www.quansight.com/). Initial creation of `meson-python` by Filipe Laíns was also funded by Quansight.

Reply:
Thanks! Implemented